prow/scripts/cluster-integration/kyma-gke-upgrade.sh

```diff
@@ -362,6 +449,25 @@ function testKyma() {
     shout "Test Kyma"
     date
     "${KYMA_SCRIPTS_DIR}"/testing.sh
+
+    shout "Test Kyma end-to-end upgrade scenarios"
+    date
+
+    helm test "${UPGRADE_TEST_RELEASE_NAME}" --timeout "${UPGRADE_TEST_HELM_TIMEOUT_SEC}"
+    testEndToEndResult=$?
+    if [ "${testEndToEndResult}" != 0 ]; then
+        echo "Helm test operation failed: ${testEndToEndResult}"
+        exit "${testEndToEndResult}"
+    fi
+
+    checkTestPodTerminated
+    executeTestResult=$?
```

Comment:
We shouldnt have that here. This will only spams our logs. Helm already waiting for test to be completed (with respection to given timeout)